index.html

```diff
@@ -22,719 +23,758 @@
     <link rel="stylesheet" href="css/bootstrap-datepicker.css">
     <link rel="stylesheet" href="css/jquery.timepicker.css">
 
-    
+
     <link rel="stylesheet" href="css/flaticon.css">
     <link rel="stylesheet" href="css/icomoon.css">
     <link rel="stylesheet" href="css/style.css">
-  </head>
-  <body>
+    <script src="https://kit.fontawesome.com/b98fb790ca.js" crossorigin="anonymous"></script>
```

Comment:
Love the font! For branding purposes, I think we should chat in slack about the 2-3 fonts we want to consistently use throughout the page and we can create CSS variables for them.  We should also probably decide on our color scheme too!